Changes

```diff
@@ -171,9 +175,11 @@ OCaml 4.05.0 (TBD):
   ListLabels, MoreLabels, StringLabels so they are compatible with
   non-labeled counterparts. Also add missing @@ocaml.deprecated attributes
   in StringLabels and BytesLabels.
-  (Roma Sokolov)
+  (Roma Sokolov, review by Gabriel Scherer, Jacques Garrigue,
+   Gabriel Radanne, Alain Frisch)
```

Comment:
It looks like there is no rule so far (I started doing the change you suggest, but I found other site that 1-indent, possibly of my past own making), so I'll just leave as it is for now. Thanks for the attention anyway!